sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/Component.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * SonarSource :: Update Center :: Common
+ * Copyright (C) 2010-2019 SonarSource SA
```

Comment:
Ha! We haven't released in 2020, apparently. You will probably need to update all file headers. I think we have a Maven plugin for that... Perhaps check with JulienL.